src/main/java/chess/controller/gamecontroller/GameAPIController.java

```diff
@@ -0,0 +1,25 @@
+package chess.controller.gamecontroller;
+
+import chess.dto.ScoreDto;
+import chess.service.ChessGameService;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GameAPIController {
+
+    private final ChessGameService chessGameService;
+
+    public GameAPIController(final ChessGameService chessGameService) {
+        this.chessGameService = chessGameService;
+    }
+
+    @GetMapping(path = "/{roomId}/status", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
    @GetMapping(path = "/{roomId}/status")
```

RestController라서 생략가능해보이네요~

Reply:
default가 APPLICATION_JSON_VALUE 이군요! 생략했습니다!